gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/subgraph/Subgraph.java

```diff
@@ -214,7 +215,8 @@ private LG edgeInducedSubgraph(LG superGraph) {
         .with(new RightSide<>()))
       .distinct(new Id<>());
 
-    return superGraph.getFactory().fromDataSets(filteredVertices, filteredEdges);
+    return superGraph.getFactory()
+      .fromDataSets(superGraph.getGraphHead(), filteredVertices, filteredEdges);
```

Comment:
Please add a check in the according tests for these operators to verify if the graph head is unchanged. As I see, Loading a graph from `FlinkAsciiGraphLoader` with `getLogicalGraph();` returns a graph which graph head has the label `_DB` ... this can be added to the expected graphs maybe